build.gradle

```diff
@@ -208,7 +209,11 @@ dependencies {
     provided "appeng:RotaryCraft:${config.rotc.version}:api"
     provided ("appeng:appliedenergistics2:${config.ae2.version}:dev") {
         exclude module: 'buildcraft'
-    }
+    }*/
+
```

Comment:
@fnuecke 
> What's the status here, i.e. do you think you can port the bits that currently require non-api classes to use the API?
You could modify some Code to use the AE2-API Interfaces, but this wouldn't work for everything, so the lack of upgradeability stays due not completely using the api. I suggest to use this for now and update the whole AE2 integration to the api by using the Open-Computer adapter or adding an network access block or something like this. For now I think this is the simplest and best solution.